corehq/apps/export/models/new.py

```diff
@@ -1075,10 +1080,11 @@ def _get_app_build_ids_to_process(domain, last_app_versions):
         return map(lambda app_build_version: app_build_version.build_id, app_build_verions)
 
     @staticmethod
-    def generate_schema_from_builds(domain, case_type, force_rebuild=False):
+    def generate_schema_from_builds(domain, app_id, case_type, force_rebuild=False):
```

Comment:
If I'm following correctly, `app_id` was added here so that the current app's state will be take into account when building the schema, is that correct?
Since case export schemas reflect the builds of *all* apps in the domain, it seems like it might be preferable to include the current state of all apps in the domain, not just the one selected when choosing the case type.